sensor_array/analysis/sample_analysis.py

```diff
@@ -20,7 +20,6 @@
 import tensorflow_probability as tfp
 
 
-# ----- General Use -----
```

Comment:
I would prefer the use of the word "reordered" over "reorganized" since then I could assume it is just functions moving around, rather than the possibility that a function was reorganized.

Reply:
Same comment.